docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1493,9 +1495,9 @@ For cases in which you need to modify the generated `ServiceBinding` to fix a na
 
 The `service key` is usually the name of the service, for example the name of the datasource, or the name of the `mongo` client. When this value is not available, the datasource type, such as `postgresql`, `mysql`, `mongo`, is used instead.
 
-To avoid naming conflicts between different types of services, the `service key` can  be prefixed with the specific type, such as `postgresql-__<person>__`.
+To avoid naming conflicts between different types of services, prefix the `service key`` with a specific datasource type, such as 'postgresql-__<person>__`.
```

Comment:
```suggestion
To avoid naming conflicts between different types of services, prefix the `service key` with a specific datasource type, such as `postgresql-__<person>__`.
```